src/otaclient_iot_logging_server/greengrass_config.py

```diff
@@ -237,6 +237,14 @@ def aws_cloudwatch_log_group(self) -> str:
             f"{self.account_id}/{self.profile}-edge-otaclient"
         )
 
+    @computed_field
+    @property
+    def aws_cloudwatch_metrics_log_group(self) -> str:
+        return (
+            f"/aws/greengrass/edge/{self.region}/"
+            f"{self.account_id}/{self.profile}-edge-otaclient-metrics"
+        )
+
```

Comment:
Suggests that also change the previous `aws_cloudwatch_log_group` property name into `aws_cloudwatch_otaclient_logs_log_group` as we have two log groups now.

And to make naming schema matching, we can also rename `aws_cloudwatch_metris_log_group` into `aws_cloudwatch_otaclient_metrics_log_group`.

Reply:
Thank you, fixed.